--- a/akregator/src/akregator_part.cpp
+++ b/akregator/src/akregator_part.cpp
@@ -133,6 +133,25 @@ namespace {
         look.setAttribute("xmlUrl","http://www.kde.org/kde-look-content.rdf");
         mainFolder.appendChild(look);
 
+        QDomElement debianFolder = doc.createElement( "outline" );
+        debianFolder.setAttribute("text","Kubuntu");
+        body.appendChild(debianFolder);
+
+        QDomElement planet = doc.createElement( "outline" );
+        planet.setAttribute("text",i18n("Planet Ubuntu"));
+        planet.setAttribute("xmlUrl","http://planet.ubuntu.com/rss20.xml");
+        debianFolder.appendChild(planet);
+
+        QDomElement fridge = doc.createElement( "outline" );
+        fridge.setAttribute("text",i18n("Ubuntu Fridge"));
+        fridge.setAttribute("xmlUrl","http://feeds.feedburner.com/ubuntu-fridge");
+        debianFolder.appendChild(fridge);
+
+        QDomElement dtimes = doc.createElement( "outline" );
+        dtimes.setAttribute("text",i18n("Kubuntu.org News"));
+        dtimes.setAttribute("xmlUrl","http://www.kubuntu.org/news/feed");
+        debianFolder.appendChild(dtimes);
+
         return doc;
     }
 }
